README.markdown

```diff
@@ -8539,6 +8542,8 @@ table. It won't share the same `ngx.ctx` with the Lua handler creating the timer
 If you need to pass data from the timer creator to the timer handler, please
 use the extra parameters of `ngx.timer.at()`.
 
+Since nginx has been confirmed that all timers have been cleaned up when exit worker is executed, all timers will no longer be executed in exit worker phase. Reference <https://github.com/nginx/nginx/blob/f02e2a734ef472f0dcf83ab2e8ce96d1acead8a5/src/os/unix/ngx_process_cycle.c#L715>.
```

Comment:
```
It's not allowed to create a timer (even a 0-delay timer) here since it runs after all timers have been processed.
```
better move this note to here?
https://github.com/openresty/lua-nginx-module/blob/e718d797cf28988251c329e247c7cd475d7e2bd9/README.markdown#exit_worker_by_lua_block

Reply:
And we can keep the reference only in the code comments.